Automattic/newspack-plugin
# Simplify the Setup Welcome screen

### All Submissions:






### Changes proposed in this Pull Request:

I'm removing un-needed content and only keep the essential one from the welcome screen.

* Remove SVG
* Reduce text
* Modify button content

__Before:__

![newspack test_wp-admin_admin php_page=newspack-setup-wizard(1280 x 720)](https://user-images.githubusercontent.com/177929/68680524-b4b1c980-0559-11ea-9d30-84517e781d51.png)

__After:__

![newspack test_wp-admin_admin php_page=newspack-setup-wizard(1280 x 720)](https://user-images.githubusercontent.com/177929/68680461-96e46480-0559-11ea-9cad-4c976aff3071.png)

### How to test the changes in this Pull Request:

1. Switch to this branch and run `npm run build:webpack`
2. (Re)start the set up of the plugin
3. Do you see the simplified welcome screen?

### Other information:
